commands/_birthdayMessage.py

```diff
@@ -3,6 +3,7 @@
 
 
 async def send_birthday_message(client, guild_id, channel_id):
+    await client.get_guild(guild_id).get_channel(channel_id).purge(limit = None, check = lambda msg: not msg.pinned)
```

Comment:
If we purge, won't the morning announcements get removed tho?

Reply:
Morning announcements wouldn't get removed with the way this is implemented right now, however I will change this so that purging is its own job that occurs at end of day to avoid any issues.